source/Government.cpp

```diff
@@ -115,29 +144,37 @@ void Government::Load(const DataNode &node)
 						grand.PrintTrace("Skipping unrecognized attribute:");
 				}
 		}
-		else if(child.Token(0) == "bribe" && child.Size() >= 2)
-			bribe = child.Value(1);
-		else if(child.Token(0) == "fine" && child.Size() >= 2)
-			fine = child.Value(1);
-		else if(child.Token(0) == "enforces" && child.HasChildren())
+		else if(key == "bribe" && hasValue)
+			bribe = add ? bribe + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(key == "fine" && hasValue)
+			fine = add ? fine + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(add && key == "enforces" && child.HasChildren())
+		{
+			enforcementZones.clear();
+			enforcementZones.emplace_back(child);
+		}
+		else if(key == "enforces" && child.HasChildren())
+		{
+			enforcementZones.clear();
 			enforcementZones.emplace_back(child);
-		else if(child.Token(0) == "enforces" && child.Size() == 2 && child.Token(1) == "all")
+		}
+		else if(key == "enforces" && child.Size() == 2 && child.Token(1) == "all")
```

Comment:
I'm aware that this is older code, but since we're here, the convention is typically to check >= the intended size instead of == the intended size.
```suggestion
		else if(key == "enforces" && child.Size() >= 2 && child.Token(1) == "all")
```
Although I think we could also now replace this with the hasValue and valueIndex variables.
```suggestion
		else if(key == "enforces" && hasValue && child.Token(valueIndex) == "all")
```